app/components/GenreCard/stories/Genre.stories.mdx

```diff
@@ -0,0 +1,29 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import GenreCard from '../index.js';
+
+<Meta title="Components|GenreCard" component={GenreCard} />
+
+# GenreCard Props Validation
```

Comment:
# means title/h1
## means subtitle/h2
so this one is the doc's header which is level1, 
and below two are subheaders in level2

# GenreCard
##GenreCard Default
##GenreCard Active